Games/types/Mafia/roles/cards/NightSurgeon.js

```diff
@@ -0,0 +1,25 @@
+const Card = require("../../Card");
+const { PRIORITY_NIGHT_TRAPPER } = require("../../const/Priority");
+
+module.exports = class NightSurgeon extends Card {
+
+    constructor(role) {
+        super(role);
+
+        this.meetings = {
+            "Surgeon": {
+                states: ["Night"],
+                flags: ["voting"],
+                action: {
+                    labels: ["save", "kill"],
+                    priority: PRIORITY_NIGHT_TRAPPER,
+                    run: function () {
+                        this.trapTarget();
```

Comment:
(if you want to be consistent with EM mechanics)

surgeon will kill a friendly visitor, like cop checking the target
the EM mechanic was that the kill only happens when target is attacked